src/lib/signature_lib/public_key.mli

```diff
@@ -112,3 +112,5 @@ val decompress_exn : Compressed.t -> t
 val compress_var : var -> (Compressed.var, _) Checked.t
 
 val decompress_var : Compressed.var -> (var, _) Checked.t
+
+val to_base58_check : t -> string
```

Comment:
NIT: can you put a comment that when serializing to `base58`, it will first compress the public key and then base58 it. When I was reading the PR, I thought it didn't compressed the public key.

Reply:
oh wow maybe my suggestion is wrong then too, I also made the same mistake!

Reply:
I could instead name the function Public_key.to_base58_check_compressed as was originally suggested in the issue. This might make things more explicit?